guides/plugins/plugins/framework/message-queue/add-message-handler.md

```diff
@@ -44,152 +44,6 @@ class SmsHandler
 }
 ```
 
-## Consuming messages
```

Comment:
There was likely a good reason to have this here. We should have a link at the bottom linking to the new message queue section

Reply:
added a link